src/labels/CustomLabels.labels

```diff
@@ -5101,21 +5101,45 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>pmtWizardMsgNoOppFound</shortDescription>
         <value>No opportunity found.</value>
     </labels>
+    <labels>
+        <fullName>pmtWizardInvalidRecords</fullName>
+        <categories>Payment Wizard</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>pmtWizardInvalidRecords</shortDescription>
+        <value>NOTE: This Opportunity has payment record(s) that are both Paid and Written Off. You must deselect one or both checkboxes.</value>
+    </labels>
+    <labels>
+        <fullName>pmtWizardScheduleNoAmount</fullName>
+        <categories>Payment Wizard</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>pmtWizardNoAmount</shortDescription>
+        <value>NOTE: This Opportunity has no amount. You must enter an Amount before you can schedule Payments.</value>
```

Comment:
Minor thing, but can you capitalize Amount in the first sentence here? Sorry I missed that before. @patrick-yan-sf